packs/slack/sensors/slack_sensor.py

```diff
@@ -35,7 +36,8 @@ def setup(self):
             msg = 'Failed to connect to the Slack API. Invalid token?'
             raise Exception(msg)
 
-        self._populate_cache(user_data=data['users'], channel_data=data['channels'])
+        self._populate_cache(user_data=self._api_call('users.list'),
```

Comment:
Hm, the whole idea was to avoid unnecessary API calls and re-use information why is already returned on `rtm.start`.

Did the API change or is it something else?